src/dataset-builder/DatasetBuilder.ts

```diff
@@ -682,6 +671,10 @@ export const DatasetBuilderView: React.FC<DatasetBuilderProps> = (props) => {
     return criteriaCount;
   };
 
+  const addSelectedCohort = (cohort: Cohort) => {
+    addSelectableObjectToGroup(cohort, defaultHeader, selectedCohorts, setSelectedCohorts);
```

Comment:
why are these called add and not just update?

Reply:
"add" is a little more precise language around the operation occurring here: We're adding a single cohort to a list of cohorts. But, if "update" is more in line with language used around Terra UI, that would also be fine.